docs/_articles/en/v3.x/release-information.md

```diff
@@ -12,6 +12,24 @@ We publish the latest Code Analyzer monthly.
 * Run `sf plugins update` to update Code Analyzer and help to the latest version.
 * Follow these [instructions](./en/v3.x/getting-started/install/#upgrade-plug-in) to update Code Analyzer
 
+## [v3.20.0](https://github.com/forcedotcom/sfdx-scanner/tree/v3.20.0) (1-10-2024)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v3.20.0...v3.19.0)
+
+### Release Summary
+
+* NEW (CodeAnalyzer): We made some updates to the RetireJS vulnerability database.
+* NEW (CodeAnalyzer): Our documentation now displays Salesforce CLI `sf-`style instead of `sfdx-`style commands. We recommend that you update your scripts and CI/CD processes to use `sf-`style commands. For more information, read [Salesforce CLI Command Reference](https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference_top.htm).
+* CHANGE (CodeAnalyzer): To prepare your solution for AppExchange security review, we created an AppExchange-specific PMD rule engine, `pmd-appexchange`. This new engine contains rules that help you identify common security review failure points, and fix them before you submit your solution.
+
+	To submit your managed package for AppExchange security review, you’re now required to scan your solution three times:
```

Comment:
Lines 25-31 feel a little in-the-weeds for a release note entry. Could we replace it with something like "To understand how `pmd-appexchange` impacts AppExchange Security Review, read [Scan Your Solution with Salesforce Code Analyzer](https://developer.salesforce.com/docs/atlas.en-us.packagingGuide.meta/packagingGuide/security_review_code_analyzer_scan.htm) in the ISVforce Guide."?

Reply:
@jfeingold35 Hmm these lines were in the draft that you reviewed yesterday. Do they just seem out of place in markdown text versus your G doc review?

Reply:
That said, we can pivot to your suggestion. If possible, let's get these sorts of suggestions & discussions in the G doc drafts before we get to the PR stage. Cool? Thanks!